src/components/course/CourseContextProvider.jsx

```diff
@@ -24,6 +24,8 @@ export const CourseContextProvider = ({
   currency,
   canOnlyViewHighlightSets,
 }) => {
+  const [formSubmissionError, setFormSubmissionError] = useState(null);
```

Comment:
might recommend naming this `externalCourseFormSubmissionError` or similar so it's a tad more explicit that it relates only to externally hosted courses (it's not relevant for non-Exec Ed courses).